test/cljam/io/util_test.clj

```diff
@@ -57,24 +59,80 @@
     "foo.bed.gz" :bed
     "foo.BED" :bed
     "foo.BED.GZ" :bed
+    "foo.gff" :gff
+    "foo.gff3" :gff
+    "foo.GFF" :gff
     "foo.wig" :wig
     "foo.WIG" :wig
     "foo.bigWig" :bigwig
     "foo.BIGWIG" :bigwig
     "foo.bw" :bigwig
     "foo.BW" :bigwig)
   (are [?dir]
-      (are [?path] (thrown? Exception (io-util/file-type (str ?dir ?path)))
-        "foo.bam.gz"
-        "foo.SAM.gz"
-        "foo.cram"
-        "foo.CRAM"
-        "foo.2bit.gz"
-        "foo.bcf.gz")
+       (are [?path] (thrown? Exception (io-util/file-type (str ?dir ?path)))
```

Comment:
I think there's a path bug. When `?dir` is bound to `"/home/bar"` and `?path` is `"foo.bam.gz"`, `(str ?dir ?path)` becomes `/home/barfoo.bam.gz`. It's not an intentional result. Replacing `str` with `cio/file` might be the fix.

Reply:
Good catch! Fixed in 112988f